### ISSUE TITLE
about training parameters

### ISSUE THREAD

**Issue by la-la-la-1** [ranakroychowdhury/TARNet#1]
Excuse me, I used the corresponding hyperparameter of the AE dataset provided by you for training, and the RMSE on the test set was 3.44, which was quite different from the 2.173 provided in your paper. May I ask how other parameters were set during your training?

**Comment by zfbz** [comment]
I also obtained similar results as follows:
300 epoch Dataset: AE, RMSE: 3.393368222170319, MAE: 2.492403030395508
600 epoch Dataset: AE, RMSE: 2.915842448443171, MAE: 2.3046317100524902
900 epoch Dataset: AE, RMSE: 3.417597775626798, MAE: 2.6471402645111084
